config.toml

```diff
@@ -1,3 +1,4 @@
+baseURL = "https://ipfs.io/blog"
```

Comment:
Anywhere that permalinks are used, yes.

I published to https://ipfs.io/ipfs/QmWAuCpPf1EE1zGpcZxLQEQT56Yke1sVGfXC8yZKwobCbi/

The links on the homepage all link to ipfs.io/blog now - which is undesirable?

I could change them to relative URLs? However the RSS will still be incorrect if you publish to ipfs.io/ipfs/QmHash..